src/test/data/JsonSerializableAddressBookTest/conflictingActivitiesAddressBook.json

```diff
@@ -0,0 +1,25 @@
+{
+  "persons": [ {
+    "name": "Alice Pauline",
+    "phone": "94351253",
+    "email": "pauline@example.com",
+    "address": "4th street"
+  } ],
```

Comment:
Since this is the json for the conflicting activities scenario, should we leave `"persons"` as an empty list, so that in case Radian or I break something with our `Person` implementation in the future, the corresponding test cases for this json won't be wrongly flagged? Cos we might be confused and search for `Activities` related bugs since the name for this is `conflictingActivities...json`

Likewise for the other json files.

But this is smth minor cos I think there are more urgent tasks, so I think its fine if we leave it as well. Just a suggestion